week-2/G-conditionals-2/exercise-3.js

```diff
@@ -9,7 +9,15 @@
 */
 
 function calculateGrade(mark) {
-
+  if (mark >= 80) {
+    return "A";
+  } else if (mark >= 60 && mark < 80) {
```

Comment:
This is correct, but we can make our code a bit simpler here. Because we check if `mark` is more than 80 in the first `if` statement, we know that if it gets to the `else if` statement then `mark` cannot be more than 80.

That means we don't need to check if `mark` is less than 80 in the `else if` statement.